covfee/server/app.py

```diff
@@ -127,6 +132,79 @@ def admin():
     )
 
 
+# admin interface
+@frontend.route("/prolific")
+def prolific():
+    # The journey_instance_id to use for redirection
+    journey_instance_url: Optional[str] = None
+
+    # We extract the Prolific academic worker unique id
+    prolific_project_id = request.args.get("project")
+    prolific_annotator_id = request.args.get("annotator")
```

Comment:
The three variables that prolific uses are:
`PROLIFIC_PID`, `STUDY_ID` and `SESSION_ID`

Reply:
thank you!!